content/github/getting-started-with-github/git-cheatsheet.md

```diff
@@ -6,7 +6,8 @@ versions:
   free-pro-team: '*'
   enterprise-server: '*'
   github-ae: '*'
+intro: Use this git how-to guide to enhance your workflow.  
 ---
-Learning all available Git commands at once can be a daunting task. Our cheat sheets provide a quick reference for the commands you'll use most often: see "[Git Cheat Sheets](https://training.github.com/)." The "Using Git" cheat sheet is available in several languages. 
+Learning all available Git commands at once can be a daunting task. Check out "[Git Cheat Sheets](https://training.github.com/)" for a quick reference to frequently used commands. The "Using Git" cheat sheet is available in several languages. 
```

Comment:
```suggestion
Learning all available Git commands at once can be a daunting task. You can use the "[Git Cheat Sheets](https://training.github.com/)" for a quick reference to frequently used commands. The "Using Git" cheat sheet is available in several languages. 
```
I made a small suggestion above to change it from "check out" because that's an idiom that doesn't always translate well into other languages.